test/unit/test_020_system.py

```diff
@@ -0,0 +1,37 @@
+import unittest
```

Comment:
We added a podman/tests/unit/system/test_system.py now to cover these. I think you need to rebase and bring your changes current with what we've landed over the last few days.